src/Java.Runtime.Environment/Java.Interop/JreRuntime.cs

```diff
@@ -7,6 +7,8 @@
 using System.Runtime.InteropServices;
 using System.Threading;
 
+[assembly: DefaultDllImportSearchPathsAttribute (DllImportSearchPath.SafeDirectories | DllImportSearchPath.AssemblyDirectory)]
```

Comment:
Same as above. (with the difference of using `<GenerateAssemblyInfo>false</GenerateAssemblyInfo>` as well)